examples/binding/t4-lysozyme/p-xylene-explicit.yaml

```diff
@@ -63,4 +63,6 @@ protocols:
 # Here we combine the system and the protocol to make an expirement
 experiments:
   system: t4-xylene
-  protocol: absolute-binding
\ No newline at end of file
+  protocol: absolute-binding
+  restraint:
+    type: null
```

Comment:
You can also just leave out `restraint` alltogether, the default value is `null`.